From b9c7730f9995bcd215ddc87271fcbf26fda13fa2 Mon Sep 17 00:00:00 2001
From: Sergei Miroshnichenko <s.miroshnichenko@yadro.com>
Date: Mon, 23 Nov 2020 14:13:59 +0300
Subject: [PATCH 56/57] powerpc/mm: Add EXPORT_SYMBOL_GPL for pmd_page

Other archs have the pmd_page() defined, for example, statically in a
header but for PowerPC this is a function to be exported.

Signed-off-by: Sergei Miroshnichenko <s.miroshnichenko@yadro.com>
---
 arch/powerpc/mm/pgtable_64.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/powerpc/mm/pgtable_64.c b/arch/powerpc/mm/pgtable_64.c
index 111111111..111111111 100644
--- a/arch/powerpc/mm/pgtable_64.c
+++ b/arch/powerpc/mm/pgtable_64.c
@@ -137,6 +137,7 @@ struct page *pmd_page(pmd_t pmd)
 	}
 	return virt_to_page(pmd_page_vaddr(pmd));
 }
+EXPORT_SYMBOL_GPL(pmd_page);
 
 #ifdef CONFIG_STRICT_KERNEL_RWX
 void mark_rodata_ro(void)

